pages/common/groff.md

```diff
@@ -0,0 +1,24 @@
+# groff
+
+> Typesetting program that reads plain text mixed with formatting commands and produces formatted output.
+> It is the GNU replacement for the `troff` and `nroff` Unix commands for text formatting.
+
+- Render a man page as plain text, and display the result:
+
+`groff -man -T utf8 {{manpage.1}}`
+
+- Render a man page using the ASCII output device, and display it using a pager:
```

Comment:
`device`? That's a bit strange. Could we not use `plugin` or `module` here? I for one associate `device` with a physical device or chip on my motherboard.

Reply:
"output device" is the term they use in the [documentation](http://man.gnu.org.ua/manpage/?1+groff#OPTIONS):

```
−T dev: Set output device to dev.

  Real devices in groff are:

  - dvi     TeX DVI format.
  - html
  - xhtml   HTML and XHTML output.
  - lbp     Canon CAPSL printers (LBP-4 and LBP-8 series laser printers).
  - lj4     HP LaserJet4 compatible (or other PCL5 compatible) printers.
  - ps      PostScript output.

  For the following TTY output devices, −T selects the output encoding:
	
  - ascii   7bit ASCII.
  - cp1047  Latin-1 character set for EBCDIC hosts.
  - latin1  ISO 8859-1.
  - utf8    Unicode character set in UTF-8 encoding.

  (...)

  The default device is ps.
```

Reply:
Ah, ok! That's really weird.

Reply:
Yeah... I guess when some of these older stuff were created in a time when naming conventions weren't as established as they are today (e.g. maybe "device" wasn't as strongly correlated to physical devices as it is today)